airflow/settings.py

```diff
@@ -21,42 +21,61 @@
 import os
 import sys
 
+from datadog import dogstatsd
 from sqlalchemy import create_engine
 from sqlalchemy.orm import scoped_session, sessionmaker
 from sqlalchemy.pool import NullPool
 
 from airflow import configuration as conf
 
 
-class DummyStatsLogger(object):
+class BaseStatsLogger(object):
 
     @classmethod
-    def incr(cls, stat, count=1, rate=1):
+    def incr(cls, stat, count=1, rate=1, tags=None):
```

Comment:
`tags` is a great name. `tags=[]` is even more explicit in what kind of type you're expecting.